x/gov/keeper.go

```diff
@@ -297,14 +315,14 @@ func (keeper Keeper) setTallyParams(ctx sdk.Context, tallyParams TallyParams) {
 func (keeper Keeper) AddVote(ctx sdk.Context, proposalID uint64, voterAddr sdk.AccAddress, option VoteOption) sdk.Error {
 	proposal, ok := keeper.GetProposal(ctx, proposalID)
 	if !ok {
-		return ErrUnknownProposal(keeper.codespace, proposalID)
+		return errors.ErrUnknownProposal(keeper.codespace, proposalID)
 	}
 	if proposal.Status != StatusVotingPeriod {
-		return ErrInactiveProposal(keeper.codespace, proposalID)
+		return errors.ErrInactiveProposal(keeper.codespace, proposalID)
 	}
 
 	if !validVoteOption(option) {
-		return ErrInvalidVote(keeper.codespace, option)
+		return errors.ErrInvalidVote(keeper.codespace, byte(option))
```

Comment:
why byte ?

Reply:
Because of the import cycle, does not effect the actual behavior(ErrInvalidVote uses it as a byte anyway)

Reply:
bump.

Reply:
If an option is not valid we cannot generate a human-readable representation of the option anyway